build.gradle

```diff
@@ -375,7 +381,14 @@ task dbLoad_psql_createData(type: Exec) {
 	commandLine "${projectDir}/mssql2pg.pl"
 }
 
-task dockerBuildImage(dependsOn: installDist, type: com.bmuschko.gradle.docker.tasks.image.DockerBuildImage) {
+task pushSqlBaseData(dependsOn: dbLoad_psql_createData, type: DockerCopyFileToContainer) {
```

Comment:
You need a similar extra task to update `task dbLoad_mssql`.

Reply:
To me it looks like the MS-SQL initialization is performed on the host, so need need to push the SQL to the container.

Reply:
Ah yes, indeed. Problem was that the `\gset` commands we need for PostgreSQL do *not* work with the ANET _dbScript_ command, so we need to call `psql`. :disappointed: